vaadin-grid-pro-flow-parent/vaadin-grid-pro-flow/src/main/resources/META-INF/resources/frontend/gridProConnector.js

```diff
@@ -17,6 +17,13 @@
   }
 
   window.Vaadin.Flow.gridProConnector = {
+    focusCustomEditor: (editor) => {
+      if (editor instanceof HTMLInputElement) {
+        editor.select();
+      } else if (editor.focusElement && editor.focusElement instanceof HTMLInputElement) {
+        editor.focusElement.select();
+      }
+    },
```

Comment:
I tried to add a `value-changed` listener to the component so that it would perform this task, but that won't work for native fields. Adding a `change` listener doesn't work either, since the value is changed programmatically and the event is not fired. So, instead, the solution was to explicitly call the method from GridPro after the value is set.

Reply:
I think doing this explicitly after setting the value makes a lot of sense. Using the `value-changed` event also sounds like that could interfere with typing.